routes/events.js

```diff
@@ -0,0 +1,15 @@
+import express from "express";
+import {
+  getEventByDepartment,
+  getEventByName,
+  deleteEvent,
+  getAllEvents,
+} from "../controllers/eventController.js";
+
+const router = express.Router();
+router.get("/:department_event", getEventByDepartment);
+router.get("/:name_event", getEventByName);
+router.get("/events", getAllEvents);
+router.delete("/", deleteEvent);
```

Comment:
The controller tries to find a eventName. Append :eventName to the end of the route